modules/storages/app/models/storages/nextcloud_storage.rb

```diff
@@ -70,6 +70,10 @@ def authenticate_via_idp?
       authentication_method == "oauth2_sso"
     end
 
+    def audience
+      nextcloud_audience
```

Comment:
It's not yet too late to rename the `nextcloud_audience` property to `audience`, making this alias unnecessary.

Reply:
the method gives us another possibility: having control about which type of storage CAN have an audience.

I have no big pain in keeping it like this for now. But open for anything.